libflux/flux/src/lib.rs

```diff
@@ -5,9 +5,8 @@ pub mod wasm;
 extern crate fluxcore;
 extern crate serde_aux;
 
-#[macro_use]
+//#[macro_use]
```

Comment:
This is now commented out, so I wonder if we ever needed it. I _suspect_ this was a holdout from rust < 2018 edition and can be removed.